Plot/src/main/java/io/deephaven/plot/FigureWidget.java

```diff
@@ -19,10 +22,18 @@ public class FigureWidget extends FigureImpl implements LiveWidget, LiveWidgetVi
     private static final long serialVersionUID = 763409998768966385L;
     private String[] validGroups;
 
-    @SuppressWarnings("WeakerAccess") // this is used in the python integration
     public FigureWidget(final FigureImpl figure) {
         super(figure);
-        figure.getFigure().consolidatePartitionedTables();
+        getFigure().consolidatePartitionedTables();
+
+        getFigure().getTableHandles().stream()
+                .map(TableHandle::getTable)
+                .filter(DynamicNode::notDynamicOrIsRefreshing)
+                .forEach(this::manage);
+        getFigure().getPartitionedTableHandles().stream()
+                .map(PartitionedTableHandle::getPartitionedTable)
+                .filter(DynamicNode::notDynamicOrIsRefreshing)
```

Comment:
This utility method doesn't work correctly as-is. PartitionedTable does not implement DynamicNode, because it doesn't make sense to `setRefreshing` or `addParentReference` on it. Maybe we need to factor out something like "MutableDynamicNode", and keep just `isRefreshing()` in `DynamicNode`.

Reply:
We should ignore this issue and flag it for fixing later.